src/components/Books.jsx

```diff
@@ -0,0 +1,9 @@
+import BookForm from './BookForm';
+
+const Bookstore = () => (
+  <div>
+    <BookForm />
+  </div>
+);
```

Comment:
- [ ] You did well here by creating this component, but I think you have missed an important requirement. We are told to create a booklist where each book has a title and author, this will require that we pass our data using props, I suggest you create another component called `Booklist` to hold the array/hardcoded data to be passed from a Books component.

![booklist](https://user-images.githubusercontent.com/53124548/229178192-16d70cdc-146a-43e9-83a4-e054f46d79c5.png)

Reply:
Thank you for the clarification.